### PR TITLE
Add schedule to volume set

### PR DESCRIPTION
**Pull request by MajidAbuRmila**

### CODE REVIEW

**Review by enk21 — CHANGES_REQUESTED** [review]
I updated the docs for the missing items.

Example doesn't work. Gives this error: │ Error: status: 400, body: {"status":400,"message":"\"spec.snapshots.schedule\" failed custom validation because snapshots cannot be scheduled more frequently than once an hour","details":[{"message":"\"spec.snapshots.schedule\" failed custom validation because snapshots cannot be scheduled more frequently than once an hour","path":["spec","snapshots","schedule"],"type":"any.custom","context":{"error":{},"label":"spec.snapshots.schedule","value":"* * 1 * 1","key":"schedule"}}],"id":"32ad8887-93c0-4c1c-9ce8-e57f065460c7"}

File: docs/resources/volume_set.md [hunk 1]
```diff
@@ -35,6 +35,9 @@ Optional:
 
 - **create_final_snapshot** (Boolean) If true, a volume snapshot will be created immediately before deletion of any volume in this set. Default: `true`
 - **retention_duration** (String) The default retention period for volume snapshots. This string should contain a floating point number followed by either d, h, or m. For example, "10d" would retain snapshots for 10 days.
+- **schedule** (String) A standard cron schedule expression used to determine when your job should execute. (i.e., `0 * * * *` Every hour).
```
Comment (Cuppojoe) [inline docs/resources/volume_set.md:38]: This should say 

(String) A standard cron schedule expression used to determine when a snapshot will be taken. (i.e., `0 * * * *` Every hour). Note: snapshots cannot be scheduled more often than once per hour.

File: docs/resources/volume_set.md [hunk 2]
```diff
@@ -58,6 +61,7 @@ Required:
 
 - **parent_id** (String) The GVC ID.
 - **used_by_workload** (String) The url of the workload currently using this volume set (if any).
+- **binding_id** (String) Uniquely identifies the connection between the volume set and its workload. Every time a new connection is made, a new id is generated (e.g., If a workload is updated to remove the volume set, then updated again to reattach it, the volume set will have a new a new binding id).
```
Comment (Cuppojoe) [inline docs/resources/volume_set.md:64]: "a new" is repeated here.

Comment (enk21) [reply docs/resources/volume_set.md]: Fixed

File: docs/resources/volume_set.md (hunk 1 above)
Comment (enk21) [reply docs/resources/volume_set.md]: Fixed.

File: docs/resources/volume_set.md [hunk 3]
```diff
@@ -35,6 +35,9 @@ Optional:
 
 - **create_final_snapshot** (Boolean) If true, a volume snapshot will be created immediately before deletion of any volume in this set. Default: `true`
 - **retention_duration** (String) The default retention period for volume snapshots. This string should contain a floating point number followed by either d, h, or m. For example, "10d" would retain snapshots for 10 days.
+- **schedule** (String) A standard cron schedule expression used to determine when a snapshot will be taken. (i.e., 0 \* \* \* \* Every hour). Note: snapshots cannot be scheduled more often than once per hour.
```
Comment (Cuppojoe) [inline docs/resources/volume_set.md:38]: Should there be backslashes in front of the *s?

Comment (Cuppojoe) [reply docs/resources/volume_set.md:38]: Also in this case, it should be e.g. not i.e.

Comment (enk21) [reply docs/resources/volume_set.md:38]: good catch. fixing

**Review by enk21 — APPROVED** [review]
I updated the example in the docs. Ready for merging.

**Pull request merged**